### ISSUE TITLE
Improve process management of sender coordination

### ISSUE THREAD

**Issue by markusrt** [markusrt/NRZMyk#132]
Currently sender coordination is only done on the front page. E.g., each sender has a month assigned where they are expected to enter and send their cultures for the previous year.

Currently this does not always work smoothly, hence manual reminders are necessary and eventually samples arrive in bulks and then it is difficult to handle the inflow.

This may be improved

- [x] Each organization has a month assigned on database level and frontpage lists organizations dynamically
- [x] Management of organizations is still done on DB-Level without user interface
- [ ] A daily job checks for data entry from an organization. If no new data is entered by the 10th day of a due month, then a daily reminder is sent to all users of that organization. This daily reminder stops once at least one new dataset is entered.
- [ ] Another daily job checks the cryo-asservation state for cultures of an organization. If some cultures are not stored in cryo-state by the 20th day of a due month, another daily reminder E-Mail to ship cultures is sent out. This reminder can be stopped by the recipient by clicking a "Yes, we send the data" link.
- [ ] Members of the owing laboratory can see the state for each organization on the front page. It lists the number of created but not stored and number of cryo archived entries. Both for the current period of that organization and for the total number of entries.

**Comment by markusrt** [comment]
@MaxAldejohann and @JuliaWall this is my proposal to improve sender chaos. Please review the issue and discuss here if you have ideas how to improve the sketched workflow. E.g., if you think the reminder should not be sent out daily or on different days on the month.

Do you know the processing time of cultures once they arrived at your laboratory. E.g. one organization has currently ~50 sentinel entries listed but they are not cryo archived yet. How long would it take for them to be processed at your end?

**Comment by MaxAldejohann** [comment]
@markusrt ok? @JuliaWall other suggestions?

**Comment by JuliaWall** [comment]
I agree with Max, one reminder at the beginning and a reminder at the end for us to look for the progress of this month sounds good. 
Can we start sending the reminders automatically soon, so that we can make use of it early next year (2023 ;) )?

**Comment by markusrt** [comment]
Closed via automation but not finished yet as above was just a partial PR

**Comment by markusrt** [comment]
Enabled with [Release v.1.0.2503.1407](https://github.com/markusrt/NRZMyk/releases/tag/v.1.0.2503.1407)
